src/aap_eda/analytics/analytics_collectors.py

```diff
@@ -0,0 +1,438 @@
+import os
+import platform
+
+import distro
+from ansible_base.resource_registry.models.service_identifier import service_id
+from django.conf import settings
+from django.db.models import Q
+from insights_analytics_collector import CsvFileSplitter, register
+
+from aap_eda.analytics.collector import AnalyticsCollector
+from aap_eda.core import models
+from aap_eda.utils import get_eda_version
+
+
+@register(
+    "config",
+    "1.0",
+    description="General platform configuration.",
+    config=True,
+)
+def config(**kwargs) -> dict:
+    install_type = "traditional"
+    if os.environ.get("container") == "oci":
+        install_type = "openshift"
+    elif "KUBERNETES_SERVICE_PORT" in os.environ:
+        install_type = "k8s"
+    return {
+        "install_uuid": service_id(),
+        "platform": {
+            "system": platform.system(),
+            "dist": distro.linux_distribution(),
+            "release": platform.release(),
+            "type": install_type,
+        },
+        # skip license related info so far
+        "eda_log_level": settings.APP_LOG_LEVEL,
+        "eda_version": get_eda_version(),
+        "eda_deployment_type": settings.DEPLOYMENT_TYPE,
+    }
+
+
+@register(
+    "activations_table",
+    "1.0",
+    format="csv",
+    description="Data on activations",
+)
+def activations_table(since, full_path, until, **kwargs):
+    activation_query = """COPY (SELECT core_activation.id,
+       core_activation.name,
```

Comment:
What about my previous comment? https://github.com/ansible/eda-server/pull/1048#pullrequestreview-2296837617

I really think we should try to avoid raw queries if we can, the maintenance will be easier and much less error prone.

Reply:
Changed to use Django ORM generated query